tutor/resources/styles/components/annotations/annotation.less

```diff
@@ -0,0 +1,113 @@
+tutor-highlight {
+  background-color: @tutor-highlight;
+}
+
+.annotater {
+  .widget {
+    background: @tutor-white;
+    border: thin solid @tutor-neutral-light;
+    font-size: 2rem;
+    padding: 1rem 0;
+    position: absolute;
+    transform: translate(calc(10px - 50%), 10px);
+    z-index: 20;
+    .no-select();
+
+    [role="button"] {
+      padding: 0 1.5rem;
+
+      &:first-child {
+        border-right: thin solid lightgray;
+      }
+    }
+  }
+
+  .arrow_box {
```

Comment:
Why does this use underscore while `.edit-box` is dashed?

Reply:
Because it was copied in from http://www.cssarrowplease.com/ 😸